Sources/Core/AWSClientRuntime/Errors/RestXMLError+AWS.swift

```diff
@@ -1,28 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-
-import ClientRuntime
-import class SmithyXML.Reader
-
-extension RestXMLError {
```

Comment:
This used to be the base RestXML error.  It's replaced by the new type that inherits from "BaseError".

The 404 customization in this type is now performed in the protocol generator's customization.